packages/client/hmi-client/src/workflow/tera-operator.vue

```diff
@@ -229,7 +229,10 @@ main > ul:empty {
 	gap: 0.25rem;
 }
 
-:deep(li:hover) {
+/* 	TODO: Now that it's supported this would be a great place to use css nesting.
+	The formatter doesn't recognize it yet though, maybe there could be a way to tweak that? 
+*/
```

Comment:
Do you mean Prettier?

Reply:
I don't think it's prettier preventing it. It seems to be a fundamental rule even after I change the formatter:
![image](https://github.com/DARPA-ASKEM/terarium/assets/28237258/1bc28059-ec46-4377-91d0-027d85913711)
This could still be addressed by prettier settings though

Reply:
You have to use an ampersand before your next selector when you use a combinator.
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting#combinators

So it will be:
```css
.toolbar > span {
  & > li {
    background-color: red;
  }
}
```

Reply:
Still doesn't work. Also you don't need to use `&` necessarily https://developer.chrome.com/articles/css-nesting/#nesting-examples.

Reply:
For normal selector, but when you use a combinator `>, +, etc` you need the ampersand.
Try to update prettier.

Reply:
I'll do the prettier upgrade in a separate PR since I would want to do even more changes to the CSS in `tera-operator.vue`